public/index.php

```diff
@@ -5,7 +5,7 @@
 
 $routes = scandir(__DIR__ . '/../app/Routes/');
 foreach ($routes as $route) {
-    if (strpos($route, '.php')) {
+    if (is_file(__DIR__ . '/../app/Routes/' . $route) && pathinfo(__DIR__ . '/../app/Routes/' . $route, PATHINFO_EXTENSION)['extension'] === 'php')) {
```

Comment:
What do you think of `if(mb_substr($route, -4, 4) === '.php')`

Reply:
It could work, except if a directory is called `directory.php/`.
What do you think of `if(is_file(__DIR__ . '/../app/Routes/' . $route) && mb_substr($route, -4, 4) === '.php')`

Reply:
Sound good.

Waiting the fix 😄